APP/model.js

```diff
@@ -71,7 +71,10 @@ WHERE
           .query("SELECT * FROM articles WHERE article_id = $1", [id])
           .then(({ rows }) => {
             if (rows.length > 0) {
-            return Promise.reject({ status: 200, msg: "Valid article ID, no comments found" });;
+              return Promise.reject({
+                status: 200,
```

Comment:
if you're rejecting a promise you shouldn't be setting it as a 200 but giving an error message. In this situation, you're right that it's a 200, but there shouldn't be an error because the request was successful, it should just return an empty array. It's good that you're thinking about these things now, but a lot of these situations will be covered when we do frontend error handling, for example if your backend only gives you back an empty array of comments, in the frontend you might display something to say no comments found